crates/ra_hir/src/krate.rs

```diff
@@ -1,6 +1,6 @@
 pub use ra_db::CrateId;
```

Comment:
This is unrelated, but perhaps a quick comment that the module name is `krate` to avoid conflicts with the `crate` keyword would be useful here.

Reply:
Not sure: we'll be using `krate` all over the place, and commenting each instance doesn't seem worthwhile.

Reply:
Yeah, I think if we just have one comment somewhere just to make it clear. It doesn't really matter though.